src/custom-element.js

```diff
@@ -945,13 +945,7 @@ export function createAmpElementProto(win, name, implementationClass) {
    * @package @final
    */
   ElementProto.getRealChildNodes = function() {
-    const nodes = [];
-    for (let n = this.firstChild; n; n = n.nextSibling) {
-      if (!isInternalOrServiceNode(n)) {
-        nodes.push(n);
-      }
-    }
-    return nodes;
+    return dom.childNodes(this, e => !isInternalOrServiceNode(e));
```

Comment:
Use `node` instead of `e` for better readability. Usually `e` refers to an event object.